frontend/server/k8s-helper.ts

```diff
@@ -156,7 +157,8 @@ export async function getTensorboardInstance(logdir: string): Promise<string> {
         viewer &&
         viewer.body &&
         viewer.body.spec.tensorboardSpec.logDir == logdir &&
-        viewer.body.spec.type == 'tensorboard'
+        viewer.body.spec.type == 'tensorboard' &&
+        viewer.body.spec.tensorboardSpec.tensorflowImage == 'tensorflow/tensorflow:' + tfversion
```

Comment:
Existing kfp deployment might have crd files which don't have this field. When upgrading to this kfp version, the afore-mentioned crd files  might trigger crd controller error.